wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -97,8 +99,52 @@ Mapsets of songs that originated from other rhythm games often borrow that game'
 - ![](/wiki/shared/diff/normal-s.png) NOVICE / NOV
 - ![](/wiki/shared/diff/hard-s.png) ADVANCED / ADV
 - ![](/wiki/shared/diff/insane-s.png) EXHAUST / EXH
+
+For **SOUND VOLTEX II -infinite infection-** songs:
+
 - ![](/wiki/shared/diff/expert-s.png) INFINITE / INF
 
+For **SOUND VOLTEX III GRAVITY WARS** songs:
+
+- ![](/wiki/shared/diff/expert-s.png) GRAVITY / GRV
+
+For **SOUND VOLTEX IV HEAVENLY HAVEN** songs:
+
+- ![](/wiki/shared/diff/expert-s.png) MAXIMUM / MXM
+- ![](/wiki/shared/diff/expertplus-s.png) HEAVENLY / HVN
+
+For **SOUND VOLTEX VIVID WAVE** songs:
+
+- ![](/wiki/shared/diff/expert-s.png) MAXIMUM / MXM
+- ![](/wiki/shared/diff/expertplus-s.png) VIVID / VVD
+
+If more than provided difficulty names are needed for large mapsets, however, using difficulty names from other versions is allowed.
+
+### maimai
+
+- ![](/wiki/shared/diff/easy-s.png) EASY
+- ![](/wiki/shared/diff/normal-s.png) BASIC
+- ![](/wiki/shared/diff/hard-s.png) ADVANCED
+- ![](/wiki/shared/diff/insane-s.png) EXPERT
+- ![](/wiki/shared/diff/expert-s.png) MASTER
+- ![](/wiki/shared/diff/expertplus-s.png) Re:MASTER
+
+### CHUNITHM
+
+- ![](/wiki/shared/diff/normal-s.png) BASIC
+- ![](/wiki/shared/diff/hard-s.png) ADVANCED
+- ![](/wiki/shared/diff/insane-s.png) EXPERT
+- ![](/wiki/shared/diff/expert-s.png) MASTER
+- ![](/wiki/shared/diff/expertplus-s.png) WORLD'S END
+
+### Ongeki
```

Comment:
not sure how relevant this is, I can't find any maps with leaderboards that use it (?)

Reply:
hmm true
might be better to add Rayark's games instead since i've seen their respective difficulty names used on the ranked maps